library/src/otx/recipe/detection/rtmdet_tiny.yaml

```diff
@@ -61,7 +61,6 @@ overrides:
     input_size:
       - 640
       - 640
-    image_color_channel: BGR
```

Comment:
Some models expect BGR data as input, others RGB. If you remove this line from the recipes, how do you know how the right color format to use to prepare the input data? Note that training on RGB and BGR is not equivalent because the pre-trained weights are generated w.r.t. input data formatted and normalized in a specific way.

Reply:
I removed this after you removed it from OTXDataModule as this broke the recipes https://github.com/open-edge-platform/training_extensions/pull/4920/commits/eafdce1ab2c6b1aa0091c05e3307101a355c53af

I did not look into it further. Was there a specific reason why you removed it from OTXDataModule? If not, I'll look into reverting it.

Reply:
I see, makes sense :D